frontend/src/store/modules/visitors/actions.js

```diff
@@ -36,6 +37,9 @@ import {getTimeByPeriod} from '@/services/periodService';
 export default {
     [CHANGE_SELECTED_PERIOD]: (context, payload) => {
         context.commit(SET_SELECTED_PERIOD, payload.value);
+        context.dispatch(FETCH_TABLE_DATA, context.state.tableData.groupedParameter);
+        context.dispatch(FETCH_LINE_CHART_DATA, context.state.tableData.groupedParameter);
+        context.dispatch(FETCH_EACH_BUTTON_DATA, context.state.buttonData.keys());
```

Comment:
it is incorrect as well. What is "keys"?

Reply:
I mean, you need to take button types from config. Did you check what the buttonData contain?